layers/+lang/latex/funcs.el

```diff
@@ -25,19 +25,33 @@
     (`lsp (spacemacs|add-company-backends
             :backends company-capf
             :modes LaTeX-mode))
-    (_    (when (configuration-layer/package-used-p 'company-auctex)
-            (spacemacs|add-company-backends
-              :backends
-              (company-auctex-macros)
-              company-auctex-symbols
-              company-auctex-environments
-              :modes LaTeX-mode))
-          (when (configuration-layer/package-used-p 'company-reftex)
-            (spacemacs|add-company-backends
-              :backends
-              company-reftex-labels
-              company-reftex-citations
-              :modes LaTeX-mode)))))
+    ('company-math (when (and (configuration-layer/package-used-p 'company-auctex) (configuration-layer/package-used-p 'company-math))
+                       (spacemacs|add-company-backends
+                         :backends
+                         company-auctex-environments
+                         company-auctex-macros
+                         company-math-symbols-unicode
```

Comment:
I don't like this design to be honest, basically I would expect that company-math is combined with company-auctex instead of adding a new backend type. Can you combine both?

Reply:
I have done this. Now by default, when someone chooses the backend company-auctex, then company-math is used. I updated the doc so as to explain how to not use company-math. When one chooses to use company-auctex without use company-math, the layer behaves as it used to with the company-auctex backend.